pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -249,8 +249,7 @@ let
         find .  -type f -name '*.lds' -print0 | xargs -0 -r chmod u-w
 
         # Keep root and arch-specific Makefiles
-        chmod u-w Makefile
-        chmod u-w arch/$arch/Makefile*
+        chmod u-w Makefile arch/"$arch"/Makefile*
```

Comment:
Why did you change `$arch` to `"$arch"`?

Reply:
Its quoted in all other places in this file. Even if I somewhat doubt people will start adding architectures with spaces in them, its good to be consistent I guess?